client/src/components/profile/components/TimeLine.js

```diff
@@ -81,9 +98,30 @@ class TimelineInner extends Component {
     }));
   }
 
+  nextPage() {
+    this.setState(state => ({
+      ...state,
```

Comment:
Why is this line needed?

Reply:
@RandellDawson If you don't mind can you please explain what the issue is, I didn't quite understand your question thats why ?

Reply:
The whole point of setState, is you can specify which properties you want to change.  There is no reason to add the `...state,` because the other properties in state will not be effected anyway.